outline.md

```diff
@@ -0,0 +1,62 @@
+* Introduction
+  - A Brief History of Open
+  - Types of Digital Public Goods
```

Comment:
For the sake of consistency, we have been referring to (in alphabetical order):
- Open AI models
- Open content
- Open data
- Open standards
- Open-source software

Unless you include a strong rationale for changing these categories, we will ask to use the existing nomenclature.

Reply:
Done and pushed to branch; thank you.